darwin/Platform.c

```diff
@@ -133,8 +136,17 @@ const MeterClass* const Platform_meterTypes[] = {
 
 int Platform_numberOfFields = 100;
 
+static void setClockTicksPerSec() {
+   errno = 0;
+   Platform_clockTicksPerSec = sysconf(_SC_CLK_TCK);
+
+   if (errno || -1 == Platform_clockTicksPerSec) {
```

Comment:
`Platform_clockTicksPerSec < 1` doesn't make much sense either; unless time runs backwards, in which case only `Platform_clockTicksPerSec == 0`would remain meaningless, but `Platform_clockTicksPerSec == -1` became ambiguous …

Reply:
Are you suggesting that I initialize it to a value of `0`, and reserve `-1` for detecting errors?

Reply:
No, more like just checking for `< 1` and treat that as an error … ;-)